### PR TITLE
 Add proper missing error message.

### PR DESCRIPTION
**Pull request by Hyperion101010**
Added a more informative error message for error check in
 binding check in mod.rs

### CODE REVIEW

File: wgpu-core/src/device/mod.rs [hunk 1]
```diff
@@ -1004,7 +1004,8 @@ impl<F: IdentityFilter<id::BindGroupId>> Global<F> {
         let bind_group_layout = &bind_group_layout_guard[desc.layout];
         let bindings =
             unsafe { slice::from_raw_parts(desc.bindings, desc.bindings_length as usize) };
-        assert_eq!(bindings.len(), bind_group_layout.bindings.len());
+        assert_eq!(bindings.len(), bind_group_layout.bindings.len(), format!("The bind group {:?} has {} bindings
```
Comment (kvark) [inline wgpu-core/src/device/mod.rs]: you don't need `format!` when inside the `assert!`

Comment (Hyperion101010) [reply wgpu-core/src/device/mod.rs]: OK no problem, I'll update and send.

Comment (Hyperion101010) [reply wgpu-core/src/device/mod.rs]: done

**Comment by kvark** [comment]
Looks like it doesn't build?

**Comment by kvark** [comment]
@Hyperion101010 sorry, rebase is needed

**Comment by Hyperion101010** [comment]
Yes i may close it for the moment.

**Pull request closed without merging**